### PR TITLE
Super long section status badge

### LINKED ISSUE #254: Super long section status badge
In SectionResults need to reduce width of badge with sectionChip

image

### PR DESCRIPTION
**Pull request by nyet-ty**
Resolve #254

### CODE REVIEW

**Comment by nyet-ty** [comment]
Снимок экрана 2023-10-11 в 11 27 03

File: src/components/sections/SectionResults.tsx [hunk 1]
```diff
@@ -28,10 +28,14 @@ const StyledRow = styled.div`
     align-items: center;
 `;
 
-const StyledTableCell = styled.div<{ interviewer?: boolean; grade?: boolean }>`
-    width: ${({ interviewer }) => (interviewer ? '40%' : '20%')};
+const StyledTableCell = styled.div<{ text?: boolean; badge?: boolean }>`
+    width: ${({ text }) => (text ? '35%' : '15%')};
```
Comment (9teen90nine) [inline src/components/sections/SectionResults.tsx:32]: That code looks fragile

**Comment by nyet-ty** [comment]
image

File: src/components/sections/SectionResults.tsx [hunk 2]
```diff
@@ -16,73 +13,50 @@ type SectionResultsProps = {
...
-                            </StyledTableCell>
-                        </StyledRow>
-                    );
-                })}
-            </StyledTable>
+            <Text size="xl">{tr('Passed sections')}</Text>
+
+            <Table gap={10} width={600}>
+                <TableRow align="center" gap={10}>
+                    <TableCell width="250px">{tr('Section')}</TableCell>
```
Comment (9teen90nine) [inline src/components/sections/SectionResults.tsx:27]: should be a const since you're copying the widths down below

**Review by 9teen90nine — CHANGES_REQUESTED** [review]

**Review by tsumo — APPROVED** [review]

**Pull request merged**